src/cards.json

```diff
@@ -114,5 +114,9 @@
   { "title": "飢渴難耐的Aili", "type": "monster", "src":"https://i.imgur.com/8vVsIYxh.jpg"},
   { "title": "戒色第100天", "type": "trap", "src":"https://imgur.dcard.tw/trx1tL8h.png"},
   { "title": "眾基首", "type": "monster", "src":"https://i.imgur.com/VieLwoz.jpg"},
-  { "title": "斯拉夫之基", "type": "monster", "src":"https://i.imgur.com/iIQTj4s.jpg"}
+  { "title": "斯拉夫之基", "type": "monster", "src":"https://i.imgur.com/iIQTj4s.jpg"},
+  { "title": "强制色色", "type": "magic", "src":"https://i.imgur.com/8hSTlYu.jpeg"},
```

Comment:
@Hertz-Hu Dogedeck already have this card.

Reply:
Yes. I'm Simplified Chinese user, I use Microsoft Pinyin Input Method to type Traditional Chinese after Ctrl + Shift + F, but the input method generated "强制" rather than "強制", so I didn't search out the dumplicated card before. By the way, my USA IP address don't have permission to visit those imgur.dcard.tw links, I visit them with the help of archive.org